openshift/bc.yml

```diff
@@ -1,24 +1,33 @@
 apiVersion: v1
 kind: Template
+  app: '${PROJECT}-${COMPONENT}'
 parameters:
+- name: PROJECT
+  displayName: Application
+  description: The name of the application project.
+  required: true
+- name: COMPONENT
+  displayName: Component
+  description: The name of the application component.
+  required: true

```

Comment:
What do you think about changing `required: true` to `value: doc-gen`? I think it would be nice to have a default in this case.

Reply:
we seed it anyway in both cases (contributor quickstarter and ods project) via tailor,.. so I would leave it like this